pkg/packager/packager.go

```diff
@@ -54,9 +50,8 @@ func (p *Packager) CreatePackages() error {
 		if _, err := os.Stat(p.paths[i]); err != nil {
 			return err
 		}
-		deepcopier.Copy(p.config).To(settings)
```

Comment:
I removed that. The linter complained because the error is not checked. I found out that it always returned an error. So I removed it. It was completely unnecessary as the settings parameter is not used by the Run method.